### ISSUE TITLE
Please introduce ebuild for binary

### ISSUE THREAD

**Issue by ostinru** [kmeaw/clickhouse-overlay#3]
Building clickhouse from sources requires 18GB (and possibly takes a lot of time).
Is it possible to add dev-db/clickhouse-bin ebuild?

**Comment by kmeaw** [comment]
Good idea, I will publish binaries somewhere and will add clickhouse-bin ebuild.
Or maybe I should make an ebuild which extracts software vendor-provided debs?

Currently it does not take a lot of time:
   clickhouse: 846 seconds average for 2 merges
(running on i5-4460@3.2GHz, ext4 on raid1 over two 7200rpm SATA drives, MAKEOPTS=-j5)

**Comment by kmeaw** [comment]
I have re-evaluated disk space usage; actually it is much lower if system boost library is used.
